src/main/java/net/earthcomputer/clientcommands/command/CDebugCommand.java

```diff
@@ -0,0 +1,57 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.components.DebugScreenOverlay;
+import net.minecraft.util.StringRepresentable;
+import org.jetbrains.annotations.NotNull;
+
+import static dev.xpple.clientarguments.arguments.CEnumArgument.*;
+import static net.fabricmc.fabric.api.client.command.v2.ClientCommandManager.*;
+
+public class CDebugCommand {
+    public static void register(CommandDispatcher<FabricClientCommandSource> dispatcher) {
+        dispatcher.register(literal("cdebug")
+            .executes(context -> executeOverlay())
+            .then(argument("type", enumArg(DebugScreenType.class))
+                .executes(CDebugCommand::execute))
+        );
+    }
+
+    private static int execute(CommandContext<FabricClientCommandSource> ctx) {
+        DebugScreenOverlay debugScreenOverlay = Minecraft.getInstance().getDebugOverlay();
+        DebugScreenType type = getEnum(ctx, "type");
```

Comment:
Please make this method take the command source and enum as parameters. That way you can do `source#getClient` instead of `Minecraft#getInstance` and make it consistent with the other commands.